workflows/sock-shop-demo/usingCMDProbe/workflow.yaml

```diff
@@ -278,6 +270,90 @@ spec:
             "kubectl apply -f /tmp/pod-memory-hog.yaml -n {{workflow.parameters.adminModeNamespace}}",
           ]
 
+    - name: install-pod-network-loss
+      inputs:
+        artifacts:
+          - name: install-pod-network-loss
+            path: /tmp/pod-network-loss.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                description:
+                  message: |
+                    Injects network packet loss on pods belonging to an app deployment
+                kind: ChaosExperiment
+                metadata:
+                  name: pod-network-loss
+                spec:
+                  definition:
+                    scope: Namespaced
+                    permissions:
+                      - apiGroups:
+                          - ""
+                          - "batch"
+                          - "litmuschaos.io"
+                        resources:
+                          - "jobs"
```

Comment:
These permissions are not updated? Can you sync them with the latest experiment CR [refer](https://github.com/litmuschaos/chaos-charts/blob/master/charts/generic/pod-network-loss/experiment.yaml)